ZFSin/zfs/module/zfs/zfs_ioctl.c

```diff
@@ -7372,7 +7372,7 @@ zfsdev_open(dev_t dev, int flags, int devtype, struct proc *p)
 #ifdef _WIN32
 	int flags = 0;
 	int devtype = 0;
-	struct proc *p = current_proc();
+	PEPROCESS p = current_proc();
```

Comment:
This is fine. But ultimately we want `struct proc` to be `PEPROCESS` - but I think perhaps the cleaner option is if we change upstream to use a bunch of porting typedefs, like a `zfs_proc_t`. I'll ask and see how receptive they are to that idea.